keyboards/moonlander/moonlander.c

```diff
@@ -276,39 +276,21 @@ led_config_t g_led_config = { {
     {  NO_LED, 64, 59, 54, 49, 44, 39 },
     {  NO_LED, NO_LED, 60, 55, 50, 45, 40 },
     {  NO_LED, NO_LED, NO_LED, 71, 70, 69, 68 },
-}, {
-    {   0,   0 }, {   0,  12 }, {   0,  25 }, {   0,  38 }, {   0,  51 },
-    {  17,   0 }, {  17,  12 }, {  17,  25 }, {  17,  38 }, {  17,  51 },
-    {  34,   0 }, {  34,  12 }, {  34,  25 }, {  34,  38 }, {  34,  51 },
-    {  51,   0 }, {  51,  12 }, {  51,  25 }, {  51,  38 }, {  51,  51 },
-    {  68,   0 }, {  68,  12 }, {  68,  25 }, {  68,  38 }, {  68,  51 },
-    {  86,   0 }, {  86,  12 }, {  86,  25 }, {  86,  38 },
-    { 105,   0 }, { 105,  12 }, { 105,  25 },
-    {  90,  55 }, { 105,  68 }, { 116,  86 }, { 116, 59 },
-
-    { 250,   0 }, { 250,  12 }, { 250,  25 }, { 250,  38 }, { 250,  51 },
-    { 233,   0 }, { 233,  12 }, { 233,  25 }, { 233,  38 }, { 233,  51 },
-    { 216,   0 }, { 216,  12 }, { 216,  25 }, { 216,  38 }, { 216,  51 },
-    { 198,   0 }, { 198,  12 }, { 198,  25 }, { 198,  38 }, { 198,  51 },
-    { 181,   0 }, { 181,  12 }, { 181,  25 }, { 181,  38 }, { 181,  51 },
-    { 163,   0 }, { 163,  12 }, { 163,  25 }, { 163,  38 },
-    { 146,   0 }, { 146,  12 }, { 146,  25 },
-    { 161,  55 }, { 161,  68 }, { 146,  86 }, { 131, 59 }
-
-}, {
-    1, 1, 1, 1, 1, 4,
-    4, 4, 4, 1, 4, 4,
-    4, 4, 1, 4, 4, 4,
-    4, 1, 4, 4, 4, 4,
-    1, 4, 4, 4, 4, 4,
-    4, 4, 1, 1, 1, 1,
-    1, 1, 1, 1, 1, 4,
-    4, 4, 4, 1, 4, 4,
-    4, 4, 1, 4, 4, 4,
-    4, 1, 4, 4, 4, 4,
-    1, 4, 4, 4, 4, 4,
-    4, 4, 1, 1, 1, 1
-} };
+}, LED_LAYOUT_moonlander(
+    {   0,   0 }, {  17,   0 }, {  34,   0 }, {  51,   0 }, {  68,   0 }, {  86,   0 }, { 105,   0 },    { 146,   0 }, { 163,   0 }, { 181,   0 }, { 198,   0 }, { 216,   0 }, { 233,   0 }, { 250,   0 },
```

Comment:
I noted that the steps in the x-coordinate are uneven. They are mostly 17, but between 86 and 105 is a step size of 19. Also, the distribution of some excess horizontal spacing is uneven: the columns between which we have a step size of 18 differ between the left and right halves.

Moreover, the midpoint (set to 125) is not halfway between 105 and 146.